kernelci/lab/lava/__init__.py

```diff
@@ -37,7 +37,6 @@ def generate(self, params, device_config, plan_config, callback_opts=None,
         base_name = params['base_device_type']
         params.update({
             'template_file': template_file,
-            'priority': self.config.priority,
```

Comment:
See my other comment below - I believe the logic to work out the priority value belongs here as it's specific to LAVA labs.